tools/json_tools/itemgroups.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+
+"""itemgroups reporter and checker"""
+
+import argparse
+import json
+import os
+import pprint
+import sys
+from pathlib import Path
+
+
+# by default, we are not interested in any of these types, since they
+# are not items
+IGNORE_TYPES = [
```

Comment:
I have a feeling that using a whitelist instead of blacklist would be better.

Reply:
I implemented this with a whitelist instead now.
I improved some of the filtering categories too.